example/main.go

```diff
@@ -48,13 +54,16 @@ func run() error {
 	p.Play()
 
 	fmt.Printf("Length: %d[bytes]\n", d.Length())
+	fmt.Printf("Duration: %v\n", d.Duration().Round(time.Second))
 	for {
 		time.Sleep(time.Second)
 		if !p.IsPlaying() {
 			break
 		}
+		fmt.Printf("\rElapsed time: %v", d.ElapsedTime().Round(time.Second))
```

Comment:
Why \r?

Reply:
Thanks to this, in the example, the elapsed time message will update on the same line - just a visually nice effect.